fdbcli/BulkDumpCommand.actor.cpp

```diff
@@ -101,14 +102,32 @@ ACTOR Future<UID> bulkDumpCommandActor(Reference<IClusterConnectionRecord> clust
 		}
 		Key rangeBegin = tokens[2];
 		Key rangeEnd = tokens[3];
-		// Bulk load can only inject data to normal key space, aka "" ~ \xff
+		// Bulk dump can only inject data to normal key space, aka "" ~ \xff
 		if (rangeBegin >= rangeEnd || rangeEnd > normalKeys.end) {
 			printUsage(tokens[0]);
 			return UID();
 		}
 		std::string remoteRoot = tokens[4].toString();
 		KeyRange range = Standalone(KeyRangeRef(rangeBegin, rangeEnd));
-		state BulkDumpState bulkDumpJob = newBulkDumpTaskLocalSST(range, remoteRoot);
+		bulkDumpJob = newBulkDumpTaskLocalSST(range, remoteRoot);
+		wait(submitBulkDumpJob(cx, bulkDumpJob));
+		return bulkDumpJob.getJobId();
+
+	} else if (tokencmp(tokens[1], "blobstore")) {
+		if (tokens.size() != 5) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		Key rangeBegin = tokens[2];
+		Key rangeEnd = tokens[3];
+		// Bulk dump can only inject data to normal key space, aka "" ~ \xff
+		if (rangeBegin >= rangeEnd || rangeEnd > normalKeys.end) {
+			printUsage(tokens[0]);
+			return UID();
+		}
+		std::string remoteRoot = tokens[4].toString();
+		KeyRange range = Standalone(KeyRangeRef(rangeBegin, rangeEnd));
+		bulkDumpJob = newBulkDumpTaskBlobstoreSST(range, remoteRoot);
```

Comment:
nit: We repeat code here. Worth taking a minute to factor out a method?

Reply:
I agree. We can do this in a separate PR later when the UI is finalized.